src/components/verification-grid/verification-grid.ts

```diff
@@ -817,54 +960,40 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     this.showingSelectionShortcuts = false;
   }
 
-  private handleSelection(selectionEvent: SelectionEvent): void {
+  private handleTileSelection(selectionEvent: SelectionEvent): void {
```

Comment:
which events trigger this callback? mouse? everything?

Reply:
Everything at the grid tile level

So alt + number selection, and click selection

Anything that involves the interaction of multiple tiles in a selection event (e.g. highlight selection or arrow selection is not handled by this

(I admit this is not the best separation of concerns)

---

The guard is duplicated in the `processSelection` helper, and is therefore not needed.

If I remove the guard, I think the event handler + add JSDoc's, it becomes a bit cleaner

<img width="723" height="251" alt="image" src="https://github.com/user-attachments/assets/b31ebdfa-5451-45a7-a7bd-6348ea06dfbd" />

Reply:
💯comment helps a lot when trying to trace the behaviours